### PR TITLE
Updating troubleshooting and custom recommendations topics

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) reorders the troubleshooting topic and adds a note to the customizing recommendation topic.

## Affected DevDocs pages

-  https://devdocs.magento.com/recommendations/troubleshooting.html
- https://devdocs.magento.com/recommendations/customize.html

### CODE REVIEW

File: src/recommendations/customize.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ However, to ensure Magento can successfully collect metrics for the recommendati
 
 Refer to the [template overview]({{ site.baseurl }}/guides/v{{ site.version }}/frontend-dev-guide/templates/template-walkthrough.html) for more information about how to modify templates in Magento.
 
-## Recommendation unit placement
+## Reorder recommendation units
 
 When you [create a recommendation](https://docs.magento.com/m2/ee/user_guide/marketing/create-new-rec.html), you specify [where you want the recommendation unit to appear](https://docs.magento.com/m2/ee/user_guide/marketing/product-recommendations.html#productrecplacement) on your page. In the Admin, you can select to place the recommendations either above or below the main content. You can, however, customize that location by editing the `*.xml` files responsible for the layout. Magento generates these files when you create a recommendation.
```
Comment (misha-kotov) [inline src/recommendations/customize.md]: "top or the bottom of the main content container" not above/below

Comment (misha-kotov) [reply src/recommendations/customize.md]: we should say "customize that location *more exactly*" or something like that to relay that you have more control/granularity with placement this way

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
running tests

**Pull request merged**